tracer/test/Datadog.Trace.TestHelpers/PublicApiTestsBase.cs

```diff
@@ -51,6 +53,26 @@ public void PublicApiHasNotChanged()
             publicApi.Should().Be(expected, "Public API should match the verified API. Update verified snapshot when the public API changes as appropriate");
         }
 
+        [Fact]
+        public void AssemblyReferencesHaveNotChanged()
+        {
+            StringBuilder sb = new();
+            foreach (var referencedAssembly in _assembly.GetReferencedAssemblies())
```

Comment:
I think we should sort these by `FullName` to ensure the snaphshot is stable

Reply:
Fixed in 9bec47c2be323b7046244b2d40d6be099abf15a3